shared/macros-oval.jinja

```diff
@@ -1,4 +1,16 @@
-{{%- macro oval_check_config_file(msg='', path='', prefix_regex='^\s*', parameter='', separator_regex='(?-i)[\s]+', value='', default_pass='', application='', match_whole_word='') -%}}
+{{#
+  High level macro which checks for a particular set of parameter plus value in a configuration file.
+  This function can take eight values:
+    - path (String): Path to the configuration file to be checked.
+    - prefix_regex (String): Regular expression to be used in the beginning of the OVAL text file content check.
+    - parameter (String): The parameter to be checked in the configuration file.
+    - value (String): The value to be checked. This can also be a regular expression (e.g: value1|value2 can match both values).
+    - separator_regex (String): Regular expression to be used as the separator of parameter and value in a configuration file.
+    - default_pass (Any String): If set, it allows the check to pass when the default configuration value is acceptable.
```

Comment:
I suggest: `Implies that the value for the parameter is the default value. If set, the check will pass if the parameter is not set in the configuration file.

Reply:
Also, `Any String` is strange. I would just say `boolean`.

Reply:
The problem here is that if I put `boolean` here it might confuse user because they would think that false would disable the option, but it will in fact enable it, because any string would activate the option. I will investigate boolean type in jinja macros

Reply:
I changed to actual boolean type and it works, now it is possible to specify `default_pass=true` without double quotes.